robot_name: "scout_1"
# imu_topic_name : "/scout_1/imu"
cloud_topic_name: "/scout_1/Cloud"
sensor_control_topic_name: "/scout_1/sensor_controller/command"

